services/gateway/server/server.go

```diff
@@ -0,0 +1,43 @@
+package server
+
+import (
+	"github.com/gin-gonic/gin"
+
+	"github.com/Drozd0f/ttto-go/services/gateway/conf"
+	"github.com/Drozd0f/ttto-go/services/gateway/server/middleware"
+	"github.com/Drozd0f/ttto-go/services/gateway/service"
+)
+
+type Server struct {
+	*gin.Engine
+	c          *conf.Config
+	service *service.Service
+}
+
+func New(c *conf.Config, service *service.Service) (*Server, error) {
+	if !c.Debug {
+		gin.SetMode(gin.ReleaseMode)
+	}
+
+	s := &Server{
+		Engine:     gin.Default(),
+		c:          c,
+		service: service,
+	}
+	if err := s.Engine.SetTrustedProxies(nil); err != nil {
```

Comment:
Not really understand for what it's here.

https://pkg.go.dev/github.com/gin-gonic/gin#readme-don-t-trust-all-proxies